node/chainSimulator/testdata/genesisSmartContracts.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
```

Comment:
I guess we can provide an empty file for this. No need to deploy SCs at genesis time as this takes a lot of time.

Reply:
will change in the next PR because also have to change the `genesis.json` file

Reply:
great